demisto_sdk/commands/upload/uploader.py

```diff
@@ -268,7 +269,13 @@ def _upload_single(self, path: Path) -> bool:
             )
             self.failed_parsing.append((path, reason))
             return False
-
+        if (
+            self.marketplace
+            and self.marketplace not in content_item.marketplaces
+            and isinstance(content_item, ContentItem)
+        ):
+            self._skipped_upload_marketplace_mismatch.append(content_item)
+            return True
```

Comment:
that's a bit misleading since the method signature it states the returned type: "bool: whether the item is uploaded successfully."
In our case the content-item was not uploaded at all.

Reply:
Done